fictadvisor-api/src/v2/utils/documentation/permission/index.ts

```diff
@@ -0,0 +1,5 @@
+import { PermissionDocumentationCheck } from './check';
+
+export const PermissionDocumentation = {
+  CHECK: PermissionDocumentationCheck,
```

Comment:
Name of the documentation property (CHECK) must be the same as corresponding controller method (CHECK_PERMISSIONS)